tests/e2e/settings-view/specs/devtools-2.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "@wdio/globals";
+import { VSCodePO } from "../../page-objects";
+
+describe("Settings view", () => {
+    describe("devtools", () => {
+        it("should pass if option is enabled", async () => {
+            const vscodePO = await VSCodePO.create();
+            const testingViewControl = vscodePO.getTestingViewControl();
+            await testingViewControl.open();
+
+            const sidebar = vscodePO.getTestingSideBar();
+            await sidebar.waitTestsRead();
+
+            const webview = await vscodePO.getWebviewByTitle("Testplane");
+            await webview.open();
```

Comment:
should switch execution context to webview in order to be able to click on `devtools` checkbox